manifests/init.pp

```diff
@@ -292,6 +292,26 @@
   String[1] $keycloak_app_name = $foreman::params::keycloak_app_name,
   String[1] $keycloak_realm = $foreman::params::keycloak_realm,
 ) inherits foreman::params {
+
+  # this Workaround is necessary, because
+  # 1. the ERB-Template cannot handle Sensitive Data (whereas EPP would do so natively), and
+  # 2. the Function postgresql::postgresql_password is not ready for Sensitive yet (Pull-Request pending)
+  $db_password_unsensitive = if $db_password =~ Sensitive {
```

Comment:
This could use the same function to unwrap

Reply:
Not necessary anymore with [this Pullrequest](https://github.com/puppetlabs/puppet/pull/8651/files).